src/content/configuration/output.md

```diff
@@ -213,6 +214,12 @@ Using hashes based on each chunks' content:
 filename: "[chunkhash].bundle.js"
 ```
 
+Using hashes generated for extracted content:
```

Comment:
Is extracted content just content generated by eg `mini-css-extract-plugin` or does it include all other content (eg JS bundles) too? How does it differ from `chunkhash`?

Reply:
chunkhash is calculated for a chunk (an entry file) while content hash is a special hash generated by eg mini-css-extract-plugin and is calculated for extracted content only not the whole chunk content. This is very useful in the case of mini-css-extract-plugin as when an extracted css is changed the css hash only is invalidated not the entire chunk. 
did not want to add all these extra info here but will create a PR and add it with an example in https://webpack.js.org/guides/caching/#output-filenames  as per your issue